### PR TITLE
Draft changes to file for Welcome Home Integrations

### PR DESCRIPTION
**Pull request by dieterrandolph**
Changes applied to the Pentaho document to make it compatible with Astra Portal Welcome Home Integrations, as per the change document. If these changes are acceptable, we can merge them in. Thanks!

If you go to the `Files changed` tab above, you can see the specific changes I made.

I'm attaching screenshots of what I'm seeing locally; it should be virtually identical to what's in Production (with the text-based icons omitted).

Screen Shot 2022-08-25 at 12 57 14 PM
Screen Shot 2022-08-25 at 12 57 25 PM
Screen Shot 2022-08-25 at 12 57 35 PM
Screen Shot 2022-08-25 at 12 57 44 PM
Screen Shot 2022-08-25 at 12 57 54 PM

### CODE REVIEW

File: docs/pages/tools/integration/pentaho.md [hunk 1]
```diff
@@ -1,112 +1,129 @@
-*Last Update {{ git_revision_date }}* 
+---
+tags: integration, middleware, python, IoT
```
Comment (dieterrandolph) [inline docs/pages/tools/integration/pentaho.md:2]: I took a stab at tags here, @gstachni -- wide open for any changes that need to be made.

Comment (gstachni) [reply docs/pages/tools/integration/pentaho.md:2]: this is great. it will be easy for us to add the right tags to each page.

Comment (dieterrandolph) [reply docs/pages/tools/integration/pentaho.md:2]: @gstachni -- the order of the tags does not matter; the Astra Portal code will sort them. Thanks!

File: docs/pages/tools/integration/pentaho.md [hunk 2]
```diff
@@ -1,112 +1,142 @@
-*Last Update {{ git_revision_date }}* 
+---
+tags: 'integration, middleware, python, IoT'
+icon: 'https://awesome-astra.github.io/docs/img/pentaho-data-integration/logo-pentaho.png'
+developer_title: 'Hitachi'
+developer_url: 'https://sourceforge.net/projects/pentaho/'
+links:
```
Comment (dieterrandolph) [inline docs/pages/tools/integration/pentaho.md:6]: @gstachni -- I took a stab at the links; please confirm when you're able.

File: docs/pages/tools/integration/pentaho.md [hunk 3]
```diff
@@ -1,112 +1,142 @@
...
 1. Go to https://downloads.datastax.com/#odbc-jdbc-drivers.
 2. Select **Simba JDBC Driver for Apache Cassandra.**
 3. Select **JDBC 4.2.**
-4. Read the license terms and accept it *(click the checkbox).*
+4. Read the license terms and accept it _(click the checkbox)._
 5. Hit the blue **Download** button.
 6. Once the download completes, **unzip** the downloaded file.
 
-### ✅ Step 2: Import Driver JAR in Pentaho
+### <span class="nosurface">✅ </span>Step 2: Import Driver JAR in Pentaho
```
Comment (dieterrandolph) [inline docs/pages/tools/integration/pentaho.md:63]: Hiding icons...

File: docs/pages/tools/integration/pentaho.md (hunk 2 above)
Comment (gstachni) [reply docs/pages/tools/integration/pentaho.md:6]: Very cool

**Review by gstachni — APPROVED** [review]

**Pull request merged**